lib/jekyll/configuration.rb

```diff
@@ -9,6 +9,7 @@ class Configuration < Hash
       # Where things are
       "source"              => Dir.pwd,
       "destination"         => File.join(Dir.pwd, "_site"),
+      "collections_dir"     => "",
```

Comment:
in `docs/_docs/configuration.md` **`collections_dir`** has a value of `.` (interpreted as current location) but its empty here.